properties/base.py

```diff
@@ -240,9 +241,11 @@ class Instance(basic.Property):
 
     info_text = 'an instance'
 
-    def __init__(self, help, instance_class, **kwargs):
+    def __init__(self, help, instance_class, serializer=None, **kwargs):
         assert isinstance(instance_class, type)
         self.instance_class = instance_class
+        if callable(serializer):
```

Comment:
this should be dealt with in the setter for `serializer`. I believe that the serializer should be named.